### ISSUE TITLE
Remove Summary Row from Resources Monitor page

### ISSUE THREAD

**Issue by JimDalton** [TransNexus/NexOSS#296]
#### Page Path
./ui/Server_Monitors_Resources

#### Description of Issue
The Summary row should not be displayed on this page.
![image](https://cloud.githubusercontent.com/assets/18315382/20486424/ac72b854-afcd-11e6-8563-bde8bd7c347b.png)

**Comment by DmitryIsakbayev** [comment]
paths.view was missing &filterKey=.*Summary.*.  Only SDReporter needed the change, NexOSS and FC pages were already updated.

**Comment by DmitryIsakbayev** [comment]
Please test

**Comment by JimDalton** [comment]
Fixed

**Comment by JimDalton** [comment]
The Summary row still appears on SDReporter Reports based on NexOSS 8.19.0
![image](https://cloud.githubusercontent.com/assets/18315382/21016182/a7804a06-bd32-11e6-9a75-e3bee8de1b1b.png)

**Comment by DmitryIsakbayev** [comment]
Good catch, but it is not the same issue. The original issue was about the Summary row in the resource page. 
Fix: Added filterKey=.*%09TriggerGroupSummary%09.* to the paths.view.

	modified:   config/pages/NexOSS-FC/09-Server/Monitors/03-Triggers/01-Fraud-Analytics.json
	modified:   config/pages/NexOSS-FC/09-Server/Monitors/03-Triggers/02-Traffic-Analyzer.json
	modified:   config/pages/NexOSS-FC/09-Server/Monitors/03-Triggers/03-QoS.json
	modified:   config/pages/NexOSS-FC/09-Server/Monitors/03-Triggers/04-Billing.json
	modified:   config/pages/NexOSS/09-Server/Monitors/03-Triggers/01-Fraud-Analytics.json
	modified:   config/pages/NexOSS/09-Server/Monitors/03-Triggers/02-Traffic-Analyzer.json
	modified:   config/pages/NexOSS/09-Server/Monitors/03-Triggers/03-QoS.json
	modified:   config/pages/NexOSS/09-Server/Monitors/03-Triggers/04-Billing.json
	modified:   config/pages/SDReporter/04-Server/Monitors/03-Triggers/01-Fraud-Analytics.json
	modified:   config/pages/SDReporter/04-Server/Monitors/03-Triggers/02-Traffic-Analyzer.json
	modified:   config/pages/SDReporter/04-Server/Monitors/03-Triggers/03-QoS.json
